doc/dev/cache-pool.rst

```diff
@@ -112,6 +113,17 @@ evict cache objects, all hit_set_count HitSets are loaded into RAM.
 Currently there is minimal benefit for hit_set_count > 1 since the
 agent does not yet act intelligently on that information.
 
+The ``min_read_recency_for_promote`` defines how many HitSets to check for the
+existence of an object when handling a read operation. The checking result is
+used to decide whether to promote the object asynchronously. Its value should be
+between 0 and ``hit_set_count``. If it's set to 0, the object is promoted anyway.
+If it's set to 1, the current HitSet is checked. And if this object is in the
+current HitSet, it's promoted. Otherwise not. For the other values, the exact
+number of archive HitSets are checked. The object is promoted if the object is
+found in any of the HitSets. ::
```

Comment:
I thought this should be s/any/all/, but this is correct.

But.. I wonder if we'd get better behavior if we made it so that hte object has to be in *all* of the hitsets?  The idea is to make it harder to promote an object... i.e. ensure that it is really hot before pulling it into the cache.  As it stands now higher values (or 0) make it easier to promote..